runtime/vm/bindnatv.cpp

```diff
@@ -958,6 +959,41 @@ resolveNativeAddress(J9VMThread *currentThread, J9Method *nativeMethod, UDATA ru
 	return rc;
 }
 
+/**
+ * Attempt to bind the nativeMethod by looking through native libraries in the classLoader specified
+ *
+ * @param[in] currentThread current J9VMThread
+ * @param[in] classLoader pointer to the classLoader, assuming not NULL
+ * @param[in] nativeMethod the JNI native method to bind
+ * @param[in] longJNI the long mangled JNI name
+ * @param[in] shortJNI the short mangled JNI name
+ * @param[in] argCount the argument count
+ * @param[in] bindJNINative non-zero if JNI natives are allowed, zero for INL-only
+ *
+ * @return J9_NATIVE_METHOD_BIND_SUCCESS on success,
+ *         J9_NATIVE_METHOD_BIND_FAIL on failure.
+ */
+static UDATA
+bindNativeClassLoader(J9VMThread *currentThread, J9ClassLoader *classLoader, J9Method *nativeMethod, char *longJNI, char *shortJNI, U_8 argCount, UDATA bindJNINative)
+{
+	/* Search each shared library in the class loader for a matching native */
+	UDATA rc = J9_NATIVE_METHOD_BIND_FAIL;
```

Comment:
This can be declared in the inner scope.

Reply:
This helper method returns `rc` value hence this is right scope.